src/components/FastMarker.tsx

```diff
@@ -156,8 +156,7 @@ const FastMarker = (props: TFastMarkerProps) => {
 
     const { overlap_y_axis = true } = props;
 
-    const maxWidth = chartNode && !overlap_y_axis ? chartNode.offsetWidth - yAxisWidth - 10 : '100%';
-
+    const maxWidth = chartNode && !overlap_y_axis ? chartNode.offsetWidth - yAxisWidth - 50 : 'calc(100% - 50px)';   
```

Comment:
is 50px the width of y-axis?

Reply:
yes, Its the width of y-axis , we getting yAxisWidth value from the chartStore as well but it is different (60px) and setting it allowing marker to hide before it is overlapped with y-axis